cfe_internal/enterprise/federation/federation.cf

```diff
@@ -885,10 +885,13 @@ bundle agent distributed_cleanup_setup
   files:
     am_superhub|am_feeder::
       "${distributed_cleanup_dir}/."
+        perms => default:mog( "700", "$(cfengine_enterprise_federation:config.transport_user)", "root" ),
```

Comment:
+1

I think this one should be backported.

Reply:
I was planning on back-porting all of this to 3.18.x where distributed cleanup is present. Not 3.15, didn't get it back that far.